### ISSUE TITLE
Update Java package name

### ISSUE THREAD

**Issue by paulhoadley** [undur/vermilingua-maven-plugin#35]
Was `ng` just a placeholder package name? Should we update this to, say, `is.rebbi.vermilingua` now you've settled on a `groupId`? To some extent this is just aesthetic, and I don't want to create busy-work, but would just take a minute in Eclipse.

**Comment by hugithordarson** [comment]
I'm still not totally sure if I like the groupId, it was kind of picked in a hurry since the artifacts had to be published fast. Any thoughts?

**Comment by paulhoadley** [comment]
I think it does the job! Presumably it's a domain name you own. What's it for? (I see that `https://rebbi.is/` does re-direct somewhere else, and I don't know what it says!)

**Comment by hugithordarson** [comment]
Yup, my domain, randomly picked as groupId. I'd like to mull it over a little. Picking names is hard...
